src/models/converters/safes.rs

```diff
@@ -37,20 +45,7 @@ impl SafeInfo {
     }
 }
 
-async fn map_modules_address_ex(
```

Comment:
Why are closures better than a method? Personally a method looks cleaner to me

Reply:
No particular reason. Just restored it to look as it used to before async/await.